YellowNotes.Api/Startup.cs

```diff
@@ -22,6 +24,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllers();
             services.AddDbContextPool<DatabaseContext>(options => 
                 options.UseMySql(Configuration.GetConnectionString("Development")));
+            services.Configure<EmailConfiguration>(Configuration.GetSection("EmailConfiguration"));
+            services.AddTransient<IEmailService, EmailService>();
```

Comment:
This can be registered as a singleton. Now the question is why. Can you figure out? ;)

Reply:
It doesn't persist any state (don't have any fields). Registering it as a singleton is a way for better performance.

Reply:
❗️